Please bring the source up-to-date.  The `unsatisfied` externals in the log concern **dlopen/dlclose/etc/**:

```
../lib/libexiv2.so.0.27.2.2: undefined reference to `dladdr'
../lib/libexiv2.so.0.27.2.2: undefined reference to `dlopen'
../lib/libexiv2.so.0.27.2.2: undefined reference to `dlclose'
../lib/libexiv2.so.0.27.2.2: undefined reference to `dlerror'
../lib/libexiv2.so.0.27.2.2: undefined reference to `dlsym'
```

The code in src/version.cpp which uses **dlopen/dlclose/etc** was changed (for v0.27.0, I think).  Instead of using the dynamic linker, Linux and Unix now read the process tables in /proc/this/that to determine the libraries which have been loaded at run-time.

On my build of exiv2 on Ubuntu 16.04, I can't see any indication that the linker needs **dlopen/dlclose/etc/** functions.

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -58,7 +58,7 @@ before_build:
     - cmd: cd envs
     - cmd: python -m virtualenv conan
     - cmd: conan/Scripts/activate
-    - cmd: python -m pip install conan==1.11.2
+    - cmd: python -m pip install conan==1.11.2 # TODO: Update to latest version (needs python 3)
     - cmd: cd ..
     - cmd: conan --version
     - cmd: conan remote add conan-bincrafters https://api.bintray.com/conan/bincrafters/public-conan

--- a/ci/install.sh
+++ b/ci/install.sh
@@ -22,7 +22,7 @@ else
 fi
 
 python --version
-pip install conan==1.11.2
+pip install conan==1.17.0
 pip install codecov
 conan --version
 conan config set storage.path=~/conanData

--- a/conanfile.py
+++ b/conanfile.py
@@ -17,7 +17,7 @@ class Exiv2Conan(ConanFile):
                       )
 
     def configure(self):
-        self.options['libcurl'].shared = True
+        self.options['libcurl'].shared = False
         self.options['libcurl'].with_openssl = True
         self.options['gtest'].shared = True
 
@@ -34,7 +34,7 @@ class Exiv2Conan(ConanFile):
                 self.requires('gtest/1.8.1@bincrafters/stable')
 
         if self.options.webready and not os_info.is_macos:
-            self.requires('libcurl/7.61.1@bincrafters/stable')
+            self.requires('libcurl/7.64.1@bincrafters/stable')
 
         if self.options.xmp:
             self.requires('XmpSdk/2016.7@piponazo/stable') # from conan-piponazo

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -197,10 +197,6 @@ if (WIN32)
 endif()
 
 if (NOT MSVC)
-    if ( UNIX AND NOT CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" )
-        target_link_libraries( exiv2lib PRIVATE ${CMAKE_DL_LIBS}) # -ldl = dynamic loader used by src/version.cpp
-    endif()
-
     if ( CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" )
         target_link_libraries( exiv2lib PRIVATE -lprocstat)
     endif()

